packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -319,6 +321,43 @@ export async function replacePlaceholderForManifests(context: MigrationContext):
   }
 }
 
+export async function replacePlaceholderForAzureParameter(
```

Comment:
maybe just call it `migrateAzureParameter`? because it's not just replace placeholders

Reply:
Will update the name in another pr to make some overall changes. Thanks!